misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -172,7 +172,8 @@ public List<String> skipList() {
                 "predeclared-module-prefix_02.bal", "object_type_def_source_44.bal", "record_type_def_source_27.bal",
                 "func_type_source_09.bal", "func_type_source_13.bal", "func_type_source_14.bal",
                 "func_type_source_15.bal", "func_type_source_16.bal", "import_decl_source_24.bal",
-                "member_access_expr_source_11.bal", "float_literal_source_08.bal", "resiliency_source_05.bal");
+                "member_access_expr_source_11.bal", "float_literal_source_08.bal", "resiliency_source_05.bal",
+                "tuple_type_source_12.bal");
```

Comment:
Formatter should be able to run on top of syntax errors as well.
Any reason we are skipping `tuple_type_source_12.bal`?

Reply:
`type T1 [int % @annot string]` is formatted to `type T1 [int% @annot string]` , should this be handled in the formatter?

Reply:
I see. We have #35240. Since `int%` get tokenized the same way as `int %` we can keep this test as `int%` as well while enabling via the formatter.

Reply:
This had to be changed to `type T1 [int%  @annot string]` (extra space between % and @annot). Does this need to be changed?

Reply:
that's okay for now.